### PR TITLE
Im down with a cold

### PR DESCRIPTION
**Pull request by Cryolian**

### CODE REVIEW

**Comment by blackmirag3** [comment]
gws

File: src/main/java/classify/textfilecode/TextFileHandler.java [hunk 1]
```diff
@@ -34,14 +34,14 @@ public static void process(ArrayList<Student> masterStudentList, Scanner in) {
     /**
      * Used to create the parent folder for a text file for Inputting
      */
```
Comment (ParthGandhiNUS) [inline src/main/java/classify/textfilecode/TextFileHandler.java:36]: Thanks for the changes

File: src/main/java/classify/data/DataWriter.java [hunk 2]
```diff
@@ -52,13 +48,12 @@ public static void writeStudentInfoFile(List <Student> list, String folderPath,
      * @param parentPath    Path file containing the path of folder we want to make the text file in
      */
     public static void createParentFileFolder(String parentPath) {
-        try{
+        try {
```
Comment (ParthGandhiNUS) [inline src/main/java/classify/data/DataWriter.java:51]: Thanks for the changes

File: src/main/java/classify/textfilecode/TextFileReader.java [hunk 3]
```diff
@@ -19,24 +19,26 @@ public static void printCurrentInputFolder(String pathnameString){
...
      * @param currentDirectory File where we are querying for the files
      */
-    private static void getCurrentFiles(File currentDirectory){
+    private static void getCurrentFiles(File currentDirectory) {
         File[] fileList = currentDirectory.listFiles();
 
-        if (fileList.length == 0 || fileList == null){
+        if (fileList.length == 0 || fileList == null) {
             System.out.println("No files in your Input Folder!\n" + 
                     "Please add some new files in the correct format!");
```
Comment (ParthGandhiNUS) [inline src/main/java/classify/textfilecode/TextFileReader.java:27]: Thanks for the changes

File: src/main/java/classify/data/DataReader.java [hunk 4]
```diff
@@ -53,7 +53,7 @@ public class DataReader {
      * @param studentFileInput  Line-by-line information of strings in the Student_Information.txt
      * @param masterStudentList The list of all students
      */
-    public static void restoreStudentList(String studentFileInput, ArrayList <Student> masterStudentList ){
+    public static void restoreStudentList(String studentFileInput, ArrayList <Student> masterStudentList ) {
 
         if (studentFileInput.trim().isBlank()) {
             return;
```
Comment (ParthGandhiNUS) [inline src/main/java/classify/data/DataReader.java:59]: Thanks for the changes

File: src/main/java/classify/data/DataHandler.java [hunk 5]
```diff
@@ -55,7 +55,7 @@ public static void readStudentInfo(ArrayList<Student> list) throws IOException{
      *
      * @param list ArrayList containing the current students to write
      */
```
Comment (ParthGandhiNUS) [inline src/main/java/classify/data/DataHandler.java:57]: Thanks for the changes

File: docs/UserGuide.md [hunk 6]
```diff
@@ -542,13 +525,34 @@ Sort complete!
 ### Process a list of students from a text file `process`
 Reads a text file in the inputFolder folder located inside the data folder.
 
-:exclamation: The text file which is being processed **MUST** follow the format shown here[insert stuff here].
+The text file which is being processed **MUST** follow the format shown here[insert stuff here].
 
-Format: `process`, press enter, `filename` (without filetype) **or** `filename.txt` (with filetype)
+Format: `process`, press enter, `filename` (without filetype) **or** `filename.txt` (with file extension)
```
Comment (ParthGandhiNUS) [inline docs/UserGuide.md:530]: Thanks for the change!

File: src/test/java/classify/textfilecode/TextFileParserTest.java [hunk 7]
```diff
@@ -39,7 +39,7 @@ public void testAddPhoneNumberValidCase(){
     }
 
     @Test
-    public void testAddPhoneNumberInvalidCase(){
+    public void testAddPhoneNumberInvalidCase() {
```
Comment (ParthGandhiNUS) [inline src/test/java/classify/textfilecode/TextFileParserTest.java:42]: Thanks for the changes

**Comment by ParthGandhiNUS** [comment]
Thanks for the changes!

**Pull request merged**